api_docs/changelog.md

```diff
@@ -20,6 +20,14 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 257**
+
+* [`GET /invites`](/api/get-invites): The stream_ids field has been added to
+  be returned for all invites.
+* [`PATCH /invites/multiuse/{invite_id}`](/api/edit-invite-link): Added support for
+  editing multiuse invites to users as admin or non admin with regards to role and
+  associated streams.
+
```

Comment:
The wording doesn't seem clear to me. Maybe we can change it to something like: 
```
Added support for editing `invite_as` and `stream_ids` fields of reusable invitation links.
```
@timabbott thoughts?